docs/src/install.md

```diff
@@ -1,57 +1,73 @@
-
-Images (and possibly some additional packages) may be all you need to manipulate images programmatically.
-However, most users will want to take one or two additional steps:
-ensuring that you can load and display images.
-
 !!! note
     People in some regions such as China might fail to install/precompile `Images` due to poor network
     status. Using proxy/VPN that has stable connection to Amazon S3 and Github can solve this issue.
 
 ## Loading your first image
 
-When testing ideas or just following along with the documentation, it can be
-useful to have some images to work with.
-The [TestImages](https://github.com/JuliaImages/TestImages.jl) package bundles several "standard" images for you.
-If you haven't already installed it, use `pkg> add TestImages`.
-
-To load one of the images from this package, say
+If this is your first time working with images in julia, it's likely that you'll need to install some
+image IO backends to load the images. The current available backends for image files are:
+
+* [ImageMagick.jl](https://github.com/JuliaIO/ImageMagick.jl) covers most image formats and has extra
+  functionality. This can be your first choice if you don't have a preference.
+* [QuartzImageIO.jl](https://github.com/JuliaIO/QuartzImageIO.jl) exposes macOS's native image IO
+  functionality to Julia. In some cases it's faster than ImageMagick, but it might not cover all your
+  needs.
+* [ImageIO.jl](https://github.com/JuliaIO/ImageIO.jl) is a new image IO backend (requires julia >=v"1.3")
+  that provides an optimized performance for PNG files. Check benchmark
+  [here](https://github.com/JuliaIO/PNGFiles.jl/issues/1)
+* [OMETIFF.jl](https://github.com/tlnagy/OMETIFF.jl) supports
+  [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.0.0/index.html#ome-tiff) files. If you don't
+  know what it is, then it is likely that you don't need this package.
+
+These backends aren't exclusive to each other, so if you're a macOS user, you can install all these
+backends. And in most cases, you don't need to directly interact with these backends, instead, we
+use the [`FileIO.jl`](https://github.com/JuliaIO/FileIO.jl) frontend. If you've installed multiple
+backends then `FileIO` will choose the most appropriate backend acoording to your file format.
+For example, if available `ImageIO` is used to load png files.
+
+Adding these gives you a basic image IO setup:
 
 ```julia
-using TestImages
-img = testimage("mandrill")
+(v1.0) pkg> add FileIO ImageMagick ImageIO
 ```
 
-If this is your first time working with images in Julia, it's likely
-that these commands will prompt you to install one or more additional
-packages appropriate for your platform; you should generally accept
-the recommendation, unless you have reasons to prefer an alternate
-solution.
+and to load an image, you can use
+
+```@example
+using FileIO
+using ImageShow # hide
+img = load(joinpath("assets", "installation", "mandrill.tiff"))
```

Comment:
From which directory should this be run? It's useful as an internal test but I am less sure it's useful to users.

Reply:
The new commit hides the real image path, instead, it shows an apparently invalid path to tell users to specify their own ones.

```julia
using FileIO
using ImageShow # hide
# specify the path to your local image file
img_path = "/path/to/image.png"
img_path = joinpath("assets", "installation", "mandrill.tiff") # hide
img = load(img_path)
```